incubator/busbar/templates/svc.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "busbar.fullname" . }}-web
+  labels:
+    app: {{ template "busbar.name" . }}-web
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
+spec:
+  selector:
+    app: {{ template "busbar.name" . }}-web
+    release: "{{ .Release.Name }}"
+    component: web
+  type: LoadBalancer
+  ports:
+    - port: 80
+      targetPort: 8080
+
+---
```

Comment:
One file per resource, please.

Reply:
Because we use a S3 based registry. The `docker-registry` is based on PVs. 

Re: one file per resource, I'm working on it! =))

Reply:
S3 was recently added to the docker-registry chart. The chart is about to be moved to stable.